test_unstructured_ingest/test-ingest-slack.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+cd "$SCRIPT_DIR"/.. || exit 1
+
+if [ -z "$SLACK_CHANNELS" ] || [ -z "$SLACK_TOKEN" ]; then
+   echo "Skipping Slack ingest test because SLACK_CHANNELS and SLACK_TOKEN env vars are not set."
+   exit 0
+fi
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+        --slack-channels "${SLACK_CHANNELS}" \
+        --slack-token "${SLACK_TOKEN}" \
+        --download-dir slack-ingest-download \
+        --structured-output-dir slack-ingest-output \
+        --start-date 2023-04-01 \
+        --end-date 2023-04-08T12:00:00-08:00
+
+if ! diff -ru slack-ingest-output test_unstructured_ingest/expected-structured-output/slack-ingest-channel; then
```

Comment:
Brand new pattern to follow to make updating fixtures easier:

https://github.com/Unstructured-IO/unstructured/blob/7b44bcd/test_unstructured_ingest/test-ingest-azure.sh#L17

Ingest is a moving target! :)